msteams-platform/concepts/design/map-use-cases.md

```diff
@@ -1,72 +1,105 @@
 author: clearab
-description: Decide how to distribute your app
+description: Choose the correct app scope
 ms.topic: conceptual
 ms.author: anclear
 ---
-# Map your use cases to teams app capabilities
 
-If you haven't already, make sure you've [considered your use cases](~/concepts/design/map-use-cases.md) carefully. You should also have a good understanding of the [extensibility points and UI elements](~/concepts/extensibility-points.md) available for your app. Once you've figured out *what* your trying to solve, and *who* you're solving it for, it is time to start thinking about *how*.
+# Map your use cases to Teams app capabilities
 
-Below you'll find some common scenarios, and a selection of extensibility points and UI elements that work well with them. It isn't intended to be an exhaustive list, just to help you think through some of the possibilities available to you and the Teams platform.
+After you have identified *who* the user is and *what* problem you will solve, it is time to decide *how* to solve the problem. The *who*, *what*, and *how* completes the process of understanding and mapping your use cases to Teams app capabilities. You need to define the scope of the app based on the responses you have received and then decide which capability is best suited to build your app.
 
-## Create, share and collaborate on items in an external system
+> [!NOTE]
+> You must have a good understanding of the [entry points and UI elements](../../concepts/extensibility-points.md) available for your app. You must also make sure that you [considered your use cases](../../concepts/design/understand-use-cases.md) carefully.
 
-App for Microsoft Teams are a great way to interact with your data, and there are a variety of integration points to choose from.
+## Choose the correct scope for your app
 
-* Messaging extensions with search commands - Search external systems and share the results as an interactive card.
+Before choosing the app scope, consider the following:
 
-* Messaging extensions with action commands - Collect information to insert into a data store or perform advanced searches.
+* An app can exist across scopes.
+* App capabilities, such as messaging extensions, follow users across scopes.
+* Users are often hesitant to add apps to Teams or channels.
+* Guest users can access content exposed in Teams or channels.
 
-* Tabs - Create embedded web experiences to view, work with, and share data.
+You can choose between personal scope and team or channel scope for your app depending on the following:
 
-* Connectors and webhooks - A simple way to push data into, and send data out of the Teams client.
+* For selecting personal scope, answer questions, such as:
+  * Are there one-on-one interactions with the app required for privacy or other reasons? For example, checking leave balance.
+  * Is there going to be collaboration between users who might not have any common Teams? For example, finding upcoming events in a company.
```

Comment:
collaboration '**_among_**' users. For example, finding upcoming org wide events in a company